tests/fixtures/config/minimal_content_types.yaml

```diff
@@ -0,0 +1,16 @@
+# Minimal ContentTypes
+
+foo:
+  name: Bars
+  singular_name: Bar
+  fields: []
+
+qux:
+  slug: corges
+  singular_slug: corge
+  fields: []
+
+# Infer fields, and determine both the `slug` and `name` get inferred from the key
+grault:
+  singular_name: Waldo
+  fields: []
```

Comment:
can this line be omitted and fields set as an empty array by default in the parser?

Reply:
Not really.. If you do, you get an exception by design, to warn the user a contenttype needs fields: 

```
In content type <code>foo</code>, no 'fields' are set. Please edit <code>contenttypes.yml</code>, and correct this.
```

Reply:
.. and before you ask.. Yes, this is tested in `testIgnoreNonsensicalFileParse()` ;-)